src/themes/bootstrap-table/bootstrap-table.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @author zhixin wen <wenzhixin2010@gmail.com>
+ * https://github.com/wenzhixin/bootstrap-table/
+ * theme: https://github.com/jgthms/bulma/
```

Comment:
Need to update `https://github.com/jgthms/bulma/`

Reply:
Its WIP, so no need to review it yet :P

Reply:
Got it